jscomp/core/js_packages_info.ml

```diff
@@ -238,7 +238,9 @@ let string_of_module_id
           else  
             begin match module_system with 
               | AmdJS | NodeJS | Es6 -> 
-                dep_package_name // dep_path // js_file
+                (* HACK - TODO Replace belt-js upstream and not here *)
+                let upd_dep_package_name = if dep_package_name = "bs-platform" then "belt-js" else dep_package_name in
```

Comment:
This really does seem like the wrong place to make the swap, and it also breaks `-global` module system users (of which I am one)

Reply:
Happy to move it somewhere else, if you have suggestions. I'll try to fix the `-global` platform.

Reply:
I wish I could help, but I sadly haven't looked at the BuckleScript code much at all.

Reply:
I also thought about this [`package_name` in `bs_version`](https://github.com/jchavarri/bucklescript/blob/proposal-bs-stdlib-package/jscomp/common/bs_version.ml#L28), that gets generated from [`build_version`](https://github.com/jchavarri/bucklescript/blob/proposal-bs-stdlib-package/jscomp/build_version.js#L40). @bobzhang would that be better? I'm unsure if that'd break other things.